docs/configuration/monitoring.md

```diff
@@ -12,32 +12,51 @@ we have all functionality available on Grafana.
 
 The simplest way to setup prometheus in your cluster is by using the
 [Prometheus-Operator](https://github.com/helm/charts/tree/master/stable/prometheus-operator)
-bundle.
+bundle. Start by adding the custom resources required for installing this bundle in your cluster.
```

Comment:
I think we need to separate documents for the "crow" way of monitoring from the Prometheus + Loki + Grafana.

Also.. I believe it would be more clear if we just went through each of the components one by one:

# 1. Install Prometheus

> One liner saying what is prometheus

> A few line on how to install

# 2. Install Loki

# 3. Install Grafana

> Including how to configure the ingress to access it....

# 4. Enable Codacy ServiceMonitors and Dashboards

Reply:
Yes, I'll make it more legible. Note that Grafana and prometheus are installed by the [same chart](https://github.com/helm/charts/tree/master/stable/prometheus-operator).

Given that the documentation for monitoring is currently not vast it is probably not too hard to follow. Having the docs for crow and prometheus+loki+grafana in the same place has the virtue of making it easier for a user to get info about the monitoring options available.

We could make it clearer by changing the text at the top of the document, and possibly inverting the order, so that the simpler and built-in solution (crow) is documented first. I'll tweak it a bit.